appyters/Achilles_Harmonizome_ETL/templates/form.j2

```diff
@@ -0,0 +1,24 @@
+{% extends "biojupies/form.j2" %}
+
+{% block head %}
+{{ super() }}
+{% endblock %}
+
+{% block style %}
+{{ super() }}
+{% endblock %}
+
+{% block body %}
+<div class="container">
+    <div class="mt-3 row align-items-center">
+        <div class="col-4">
+            <img src="/static/harmonizome_logo.svg" style="width:100px" class="img-fluid float-right">
```

Comment:
Please use `url_for('static', filename='harmonizome_logo.svg')`, I'm under the strong impression this path will not work in production. That's because your application is mounted at `https://appyters.maayanlab.cloud/YourApp/` using the slash here makes it go to `https://appyeters.maayanlab.cloud/static/harmonizome_logo.svg`. This is why we use `url_for`, irrespective of how the application is mounted, it will resolve the correct url.